examples/crt/main.go

```diff
@@ -0,0 +1,290 @@
+				if (obj.Y + obj.H) > 320 {
+					obj.Y = 319 - obj.H
+					obj.VY = 1 //subtle bounce effect
+				}
+				if obj.VX > 0 {
+					obj.Flipped = false
+				} else if obj.VX < 0 {
+					obj.Flipped = true
+				}
+			}
+			obj.X += int(obj.VX)
+			obj.VX -= (obj.VX / 8)
+			obj.Y += int(obj.VY)
+			obj.VY -= (obj.VY / 8)
+		}
+	}
+	return nil
+}
+
+// Draw draws the game screen.
+// Draw is called every frame (typically 1/60[s] for 60Hz display).
+func (g *Game) Draw(screen *ebiten.Image) {
+	g.Time++
+	// Write your game's rendering.
+	bgopts := &ebiten.DrawImageOptions{}
+	g.Offscreen.DrawImage(bgImage, bgopts)
+	i, _ := getPlayerIndex(g.Objects)
+	p := g.Objects[i]
+	DrawOffsetX := -(p.VX / 640) * 200
+	DrawOffsetY := -(p.VY / 480) * 50
+	for z := 0; z < 3; z++ {
+		for _, obj := range g.Objects {
+			if !obj.Hidden {
+				if obj.Z == z {
+					obj.Draw(g.Offscreen, DrawOffsetX, DrawOffsetY)
+				}
+			}
+		}
+	}
+	text.Draw(g.Offscreen, "COLLECT EBITEN!", mainFont, int(DrawOffsetX+340+math.Cos(float64(g.Time/10))*2), int(DrawOffsetY+40+math.Sin(float64(g.Time/10))*2), color.White)
+	text.Draw(g.Offscreen, fmt.Sprintf("%d", g.Score), mainFont, int(DrawOffsetX+50+math.Cos(float64(g.Time/10))*2), int(DrawOffsetY+40+math.Sin(float64(g.Time/10))*2), color.White)
+	opts := &ebiten.DrawImageOptions{}
+	opts.GeoM.Translate(DrawOffsetX+math.Cos(float64(g.Time/10))*2, DrawOffsetY+math.Sin(float64(g.Time/10))*2)
+	g.Offscreen.DrawImage(ebitenImage, opts)
+	// These last few lines of the draw loop are of concern if you are looking to execute the crt shader.
+	op := &ebiten.DrawRectShaderOptions{}
+	op.Uniforms = map[string]interface{}{
+		"Time": float32(g.Time) / 60,
+	}
+	op.Images[0] = g.Offscreen
+	screen.DrawRectShader(640, 480, ntscShader, op)
+	g.Offscreen.Clear()
+}
+
+// Layout takes the outside size (e.g., the window size) and returns the (logical) screen size.
+// If you don't have to adjust the screen size with the outside size, just return a fixed size.
+func (g *Game) Layout(outsideWidth, outsideHeight int) (screenWidth, screenHeight int) {
+	return 640, 480
+}
+
+var (
+	gopherImage *ebiten.Image
+	floorImage  *ebiten.Image
+	ebitenImage *ebiten.Image
+	bgImage     *ebiten.Image
+	ntscShader  *ebiten.Shader
+	mainFont    font.Face
+)
+
+func main() {
+	rand.Seed(time.Now().UnixNano())
+	tt, err := truetype.Parse(fonts.PressStart2P_ttf)
+	if err != nil {
+		log.Fatal(err)
+	}
+	const dpi = 72
+	mainFont = truetype.NewFace(tt, &truetype.Options{
+		Size:    20,
+		DPI:     dpi,
+		Hinting: font.HintingFull,
+	})
+	imgtemp, _, err := ebitenutil.NewImageFromFile("gopher.png")
+	if err != nil {
+		log.Fatal(err)
+	}
+	gopherImage = imgtemp
+	imgtemp, _, err = ebitenutil.NewImageFromFile("floor.png")
+	if err != nil {
+		log.Fatal(err)
+	}
+	floorImage = imgtemp
+	imgtemp, _, err = ebitenutil.NewImageFromFile("ebiten.png")
+	if err != nil {
+		log.Fatal(err)
+	}
+	ebitenImage = imgtemp
+	imgtemp, _, err = ebitenutil.NewImageFromFile("bg.png")
```

Comment:
Instead of reading resources from files, please embed them. See the other examples and what they do.